github/util.py

```diff
@@ -70,15 +70,11 @@ def create_or_update_file(
             branch = self.default_branch
 
         try:
-            contents = self.repository.file_contents(
-                path=file_path,
-                ref=branch,
-            )
+            decoded_contents = self.retrieve_text_file_contents(file_path=file_path, branch=branch)
```

Comment:
def retrieve_file(file_path, branch):
  \# retrieve file via self.repository..... and return it

def retrieve_file_contents(.....):
  contents = self._retrieve_file(...)
  return contents.decode(...)